sources/node-typescript/src/adapters/utils.ts

```diff
@@ -122,6 +128,9 @@ export function createCancelableAction<T>(
 
     cancelableRequest[responseType]()
       .then(function (response) {
+        if (typeof response === 'string' && response.includes('KalturaAPIException')) {
+          response = safeJsonParse(response) || response
```

Comment:
Can the response be an integer `0`?
Maybe it would be safer to do it like this:
```suggestion
          response = safeJsonParse(response) ?? response
```